airflow/airflow.go

```diff
@@ -118,14 +126,21 @@ func Init(path, airflowImageName, airflowImageTag string) error {
 		".astro/dag_integrity_exceptions.txt":  "# Add dag files to exempt from parse test below. ex: dags/<test-file>",
 	}
 
-	// Initailize directories
-	if err := initDirs(path, dirs); err != nil {
-		return errors.Wrap(err, "failed to create project directories")
-	}
+	if template != "" {
```

Comment:
how would the custom runtime version work with this new template logic?

Reply:
Template based astro projects will always have latest runtime version as we are fetching these files from `astronomer/templates` repo. I have included a note in description that `airflow-version` flag will not be respected when using `--template` flag.